src/common/lints/lintSettings.ml

```diff
@@ -16,6 +16,7 @@ type sketchy_null_kind =
 type lint_kind =
   | SketchyNull of sketchy_null_kind
   | UntypedTypeImport
+  | UnknownProperty
```

Comment:
We need a better name then `UnknownProperty` since [we do error for unknown properties](https://flow.org/try/#0MYewdgzgLgBCBGArAXDA3gMxCVYCuAtvAKYBOAvjALzpY4wAsATOQNwBQ7CiAdPAJZgA5qxgB6MTACipUiFKd+GGAApufQUICU6ShJgA5ELACGYGGTkKgA). We don’t error specifically in branch tests. (Such as `if ()` or `while ()`.) How about `UnknownPropertyInBranchTest` and `unknown-property-in-branch-test`?